src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureFlagOptions.cs

```diff
@@ -27,7 +38,26 @@ public class FeatureFlagOptions
         /// <summary>
         /// The time after which the cached values of the feature flags expire.  Must be greater than or equal to 1 second.
         /// </summary>
-        public TimeSpan CacheExpirationInterval { get; set; } = RefreshConstants.DefaultFeatureFlagsCacheExpirationInterval;
+        [Obsolete("The CacheExpirationInterval property will be deprecated in a future release. Please use the new `SetFeatureFlagRefreshInterval` method instead. " +
```

Comment:
Method has been renamed to `SetRefreshInterval`

Reply:
Can we use `nameof(SetRefreshInterval)` to avoid errors like this in future?

Reply:
> property will be deprecated in a future release

Probably just terminology, but it sounds like the API is still not deprecated. How do you like this?

... property is deprecated and will be removed in a future release...